packages/flutter/lib/src/material/popup_menu.dart

```diff
@@ -898,6 +920,16 @@ class PopupMenuButton<T> extends StatefulWidget {
   /// but doesn't currently have anything to show in the menu.
   final bool enabled;
 
+  /// If provided, the shape used for the menu.
+  ///
+  /// If this is null, then popupMenuTheme.shape is used.
+  final ShapeBorder shape;
+
+  /// If provided, the color used for the menu.
+  ///
+  /// If this is null, then popupMenuTheme.color is used.
```

Comment:
```suggestion
  /// If this is null, then [PopupMenuThemeData.color] is used.
```

Reply:
Is there a default color for the menu if neither are specified?

Reply:
If these are null, then Theme.of(context).cardColor is used. If that is also null, then [ThemeData.cardTheme.color] is used. If that is also null, then [ThemeData.cardColor] is used.